### ISSUE TITLE
Creator Labels are no longer connected

### ISSUE THREAD

**Issue by cam156** [psu-libraries/cho#949]
The labels for role and creator are no longer attached to their input field.  I 

## Wave errors for no label and orphaned label
![Screen Shot 2019-04-10 at 11 09 48 AM](https://user-images.githubusercontent.com/1599081/55890698-2d464700-5b81-11e9-9869-a7b97a84e097.png)

## inspection showing label-for and actual id are not the same
![Screen Shot 2019-04-10 at 11 08 44 AM](https://user-images.githubusercontent.com/1599081/55890705-2e777400-5b81-11e9-8efe-625fe2173d8f.png)

**Comment by mtribone** [comment]
When hovering over the "add another x" with the mouse, the cursor doesn't turn into a hand indicating that it is a clickable link. Also there aren't any hover or active states when clicking the "add" link.
